bench/cli.py

```diff
@@ -240,3 +242,17 @@ def _chdir(*args, **kwargs):
 		return f(*args, **kwargs)
 
 	os.chdir = _chdir
+
+
+def setup_exception_handler():
+	from traceback import format_exception
+
+	def handle_exception(exc_type, exc_info, tb):
+		print("".join(generate_exc(exc_type, exc_info, tb)))
+
+	def generate_exc(exc_type, exc_info, tb):
+		for t in format_exception(exc_type, exc_info, tb):
+			if "/click/" not in t:
+				yield t
```

Comment:
Wondered this too. Perhaps this change should only come into effect for certain Bench Exceptions? Would appreciate thoughts from more users on this.

Should only click be excluded or are there more packages that add to the noise, which commands are more susceptible to this?

Reply:
let's revert this for now? - bunch of additional frames in traceback don't really hurt that much as opposed to if some breakages goes unnoticed? :eyes:

Reply:
The goal was to get rid of the noise which leads to confusion for newbies and non-tech users. One part of this was to print the failing command rather than a bare exception. The other is to not show an overwhelming amount of context in case of failure.

click is pretty robust - I've not seen breakages in ~5 years of using it. So I am more confident about excluding it from the traceback. I'd rather not see a CLI showing tracebacks like this at all. So, developing in that direction, I think we can apply this traceback filtering conditionally - exclude frames from internals in case "X" exception was raised...in this case - `CommandFailedError`

Reply:
also, this would also ignore tracebacks from a folder named "click" :P